### PR TITLE
Branch manager handbook updates

### PR DESCRIPTION
**Pull request by idealhack**
continue the previous work, closes https://github.com/kubernetes/sig-release/pull/604 and fixes https://github.com/kubernetes/sig-release/issues/573

this is to address the review comments, other changes can be follow-ups.

/cc @hoegaarden

### CODE REVIEW

File: release-team/role-handbooks/branch-manager/README.md [hunk 1]
```diff
@@ -28,13 +28,29 @@
 * [Non-release-workflow-specific, generic gcbmgr documentation](https://github.com/kubernetes/release/blob/master/README.md)
 * [Patch release notes from exercising the new gcbmgr flow](https://docs.google.com/document/d/1x-GQDZpKk3WajtSnO0axDazE9Xs2mOSVgjziIuTWNO0/edit)
 
+# Overview
+The release branch manager is responsible for cutting a version of [Kubernetes](https://github.com/kubernetes/kubernetes/releases). Each release is a three month cycle where as branch manager:
+1. You will cut releases of v1.X as specified on the Timeline in `sig-release/releases/release-1.X/README.md`
+1. Participate in weekly release team meetings
+1. Fast-forward the release branch on a daily basis after it's created on week 6
```
Comment (hoegaarden) [inline release-team/role-handbooks/branch-manager/README.md]: nit: Is it always week 6? I'd rather say something like:

```suggestion
1. Fast-forward the release branch on a daily basis as soon as the release branch `release-1.X` [has been created](#branch-creation).
```

File: release-team/role-handbooks/branch-manager/README.md [hunk 2]
```diff
@@ -28,13 +28,29 @@
...
+1. Commit about approximate 8 hours or less in a week of your time
+1. You will also get to select shadows and guide them in preparation to lead the next release cycle.
```
Comment (hoegaarden) [inline release-team/role-handbooks/branch-manager/README.md]:
```suggestion
1. You will also get to select shadows and guide them in preparation to become section leads for the next release cycle.
```

File: release-team/role-handbooks/branch-manager/README.md [hunk 3]
```diff
@@ -28,13 +28,29 @@
...
+
+For example, as branch manager of v1.15, the release cut dates as specified in the Timeline can be found at [`sig-release/releases/release-1.15/README.md`](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.15). The bulk of your time commitment is during release cut days.
+
+## Minimum Skills
+* Familiarity with basic Unix commands and able to debug Shell scripting.
```
Comment (hoegaarden) [inline release-team/role-handbooks/branch-manager/README.md]:
```suggestion
* Familiarity with basic Unix commands and able to debug shell scripts.
```

File: release-team/role-handbooks/branch-manager/README.md [hunk 4]
```diff
@@ -177,6 +196,9 @@ Releasing has multiple phases.  In the first phase the non-public build artifact
 * `./release-notify v1.12.0-alpha.1`
 * `./release-notify v1.12.0-alpha.1 --nomock --mailto=${{YOUREMAILADDRESS}}[a][b]`
 
+**Note**
```
Comment (hoegaarden) [inline release-team/role-handbooks/branch-manager/README.md]: nit: isn't this the same as already in https://github.com/kubernetes/sig-release/pull/619/files#diff-52866a7ca0c8666623044eeb11e59015R185 ?

**Comment by hoegaarden** [comment]
I added some nits / suggestions, but I don't want you to block on them -- remove the hold if you feel like.

/lgtm
/approve
/hold

File: release-team/role-handbooks/branch-manager/README.md (hunk 4 above)
Comment (idealhack) [reply release-team/role-handbooks/branch-manager/README.md]: I feel this too, let's drop it for now.

**Comment by idealhack** [comment]
@hoegaarden thanks for the review, I think I've addressed them all :)

**Comment by hoegaarden** [comment]
Closes: #604
/cc @Bubblemelon 
/hold cancel
/lgtm

**Pull request merged**